plugins/woocommerce-admin/client/products/product-page.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { Editor } from '@woocommerce/product-editor';
+import { Product } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import './product-page.scss';
+import './fills';
```

Comment:
I'm not sure we still need to import the fills here, since they're all for the old version and I don't believe we plan to use them here at all.

Reply:
Good catch! It's done now.